src/WebClient.spec.js

```diff
@@ -191,6 +191,19 @@ describe('WebClient', function () {
       });
     });
 
+    it('should return platform error if 200 request is not ok', function () {
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(200, { ok: false });
```

Comment:
this response isn't sufficient to test the code. there should be an `error` property in the response, the way the actual API would provide one.

Reply:
taken care of in latest commit; added the property `error: 'bad error'` to line 197's test response object